.env.example

```
@@ -0,0 +1,52 @@
+# ── OAuth2 (Kakao) ────────────────────────────
+# 카카오 개발자센터에서 앱 키 발급
+# https://developers.kakao.com
+KAKAO_CLIENT_ID=your_kakao_client_id
+KAKAO_CLIENT_SECRET=your_kakao_client_secret
+
+# ── AWS S3 ────────────────────────────────────
+# AWS IAM에서 액세스 키 발급 (로컬 개발 시 필수)
+# https://console.aws.amazon.com/iam
+AWS_ACCESS_KEY_ID=your_aws_access_key_id
+AWS_SECRET_ACCESS_KEY=your_aws_secret_access_key
+AWS_REGION=ap-northeast-2
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AWS_REGION key should go before the AWS_SECRET_ACCESS_KEY key

(UnorderedKey)